index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <h1>Hello Microverse!</h1>
```

Comment:
- [ ] I would suggest you to use class name for the `<h1>` tag element, Classes allow CSS and JavaScript to select and access specific elements via the class selectors. e.g ` <h1 class="header">Hello Microverse</h1>`
- [ ] Please kindly update your readme file  with MV readme file  template acoording to the reqirement [redme template }(https://github.com/microverseinc/readme-template/blob/master/README.md)